src/settings.rs

```diff
@@ -1,9 +1,17 @@
+// Copyright 2021-2022 Leafish Developers
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
 use crate::console;
 use crate::console::CVar;
 use std::marker::PhantomData;
 use winit::event::VirtualKeyCode;
 
-pub const R_MAX_FPS: console::CVar<i64> = console::CVar {
+pub const R_MAX_FPS: console::CVar<u8> = console::CVar {
```

Comment:
Wouldn't making this an u8 limit the max fps to 255?

Reply:
Correct.

I do not believe anyone needs more than 120 FPS _soft_-cap, and never saw more than 200 cap in games. The byte size, however, covers even 240 Hz cases. But at such speeds it is **highly undesirable** to have a soft-cap at all. It must be otherwise possible to set it to 0 (uncapped FPS) and use Vsync which is much more preferable because it also properly balances out GPU load. Soft-capping itself introduces latency which matters at such speeds.

Furthermore, I am having troubles imagining a user-friendly slider giving ability to select between 65535 values 😅

Reply:
Fair enough and yeah that would be kinda hard :sweat_smile:

Reply:
Could we just leave it at u16 for the time being? we can change it later if we decide, that u8 is better, but u16 gives us the freedom to define a limit which we want (we don't have to provide a slider for the entirety for u16 values we could i.e cap it at 500 or 1000 or even less if we wanted, we just have some freedom which i think is important)

Reply:
I will do as you ask.